bears/image/ImageCompressionBear.py

```diff
@@ -0,0 +1,96 @@
+import os
+import optimage
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from dependency_management.requirements.AnyOneOfRequirements import (
+    AnyOneOfRequirements)
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+from dependency_management.requirements.ExecutableRequirement import (
+    ExecutableRequirement)
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coala_utils.FileUtils import create_tempfile
+
+
+class ImageCompressionBear(LocalBear):
+    """
+    Checks for possible optimizations for JPEGs and PNGs
+
+    See https://github.com/sk-/optimage
+    """
+    LANGUAGES = {'Image'}
+    REQUIREMENTS = {
+        PipRequirement('optimage', '0.0.1'),
+        AnyOneOfRequirements([
+            DistributionRequirement(apt_get='jpegoptim',
+                                    portage='jpegoptim',
+                                    xbps='jpegoptim'),
+            ExecutableRequirement('jpegoptim')
+        ]),
+        AnyOneOfRequirements([
+            DistributionRequirement(apt_get='libjpeg-progs',
+                                    portage='libjpeg-turbo',
+                                    xbps='libjpeg-turbo-tools'),
+            ExecutableRequirement('jpegtran')
+        ]),
+        AnyOneOfRequirements([
+            DistributionRequirement(apt_get='pngcrush',
+                                    portage='pngcrush',
+                                    xbps='pngcrush'),
+            ExecutableRequirement('pngcrush')
+        ]),
+        AnyOneOfRequirements([
+            DistributionRequirement(apt_get='optipng',
+                                    portage='pngcrush',
+                                    xbps='pngcrush'),
+            ExecutableRequirement('optipng')
+        ]),
+        AnyOneOfRequirements([
+            DistributionRequirement(apt_get='zopfli',
```

Comment:
Zopflipng wouldn't get installed on Ubuntu 16.04 which travis runs. by `apt-get zopfli`.

To install zopflipng:
```
git clone https://github.com/google/zopfli.git
cd zopfli
make zopflipng
install -Dm755 zopflipng /usr/local/bin/zopflipng
```